numba_dppy/tests/test_dpnp_functions.py

```diff
@@ -267,6 +264,732 @@ def f(a):
+                        self.assertTrue(result >= 0.0)
+                    else:
+                        final_result = result.ravel()
+                        self.assertTrue(final_result.all() >= 0.0)
+                    self.assertIn("dpnp implementation", got_gpu_message.getvalue())
+        set_dpnp_debug(None)
+
+    def test_standard_normal(self):
+        from numba.tests.support import captured_stdout
+
+        @njit
+        def f(size):
+            res = np.random.standard_normal(size)
+            return res
+
+        set_dpnp_debug(1)
+        with captured_stdout() as got_gpu_message:
+            with dpctl.device_context("opencl:gpu"):
+                for size in self.sizes:
+                    result = f(size)
+                    if np.isscalar(result):
+                        self.assertTrue(result >= 0.0)
+                    else:
+                        final_result = result.ravel()
+                        self.assertTrue(final_result.all() >= 0.0)
+                    self.assertIn("DPNP implementation", got_gpu_message.getvalue())
+        set_dpnp_debug(None)
+
+    @unittest.skip(
+        "TypeError: dpnp_random_impl() got an unexpected keyword argument 'low'"
+    )
+    def test_uniform(self):
+        from numba.tests.support import captured_stdout
+
+        @njit
+        def f(low, high, size):
+            res = np.random.standard_normal(low=low, high=high, size=size)
+            return res
+
+        low, high = -1.0, 0.0
+        set_dpnp_debug(1)
+        with captured_stdout() as got_gpu_message:
+            with dpctl.device_context("opencl:gpu"):
+                for size in self.sizes:
+                    result = f(low, high, size)
+                    if np.isscalar(result):
+                        self.assertTrue(result >= low)
+                        self.assertTrue(result < high)
+                    else:
+                        final_result = result.ravel()
+                        self.assertTrue(final_result.all() >= low)
+                        self.assertTrue(final_result.all() < high)
+                    self.assertIn("DPNP implementation", got_gpu_message.getvalue())
+        set_dpnp_debug(None)
+
+    def test_weibull(self):
+        from numba.tests.support import captured_stdout
+
+        @njit
+        def f(a, size):
+            res = np.random.weibull(a, size)
+            return res
+
+        a = 5.0
+        set_dpnp_debug(1)
+        with captured_stdout() as got_gpu_message:
+            with dpctl.device_context("opencl:gpu"):
+                for size in self.sizes:
+                    result = f(a, size)
+                    if np.isscalar(result):
+                        self.assertTrue(result >= 0.0)
+                    else:
+                        final_result = result.ravel()
+                        self.assertTrue(final_result.all() >= 0.0)
+                    self.assertIn("dpnp implementation", got_gpu_message.getvalue())
```

Comment:
Sometimes we see catch `"dpnp implementation"`.

Reply:
This should be consistent, I agree. The prints are generated inside each implementation and hence can be different but that does not mean it should be different.